content/ko/docs/tasks/access-application-cluster/web-ui-dashboard.md

```diff
@@ -5,7 +5,7 @@ weight: 10
 card:
   name: tasks
   weight: 30
-  title: Use the Web UI Dashboard
+  title: 웹 UI 대시 보드 사용
```

Comment:
```suggestion
  title: 웹 UI 대시보드 사용
```
It is better to translate it into the single word according to the [Korean l10n glossary](https://kubernetes.io/ko/docs/contribute/localization_ko/#%ED%95%9C%EA%B8%80%ED%99%94-%EC%9A%A9%EC%96%B4%EC%A7%91).